### PR TITLE
Add deprecation warning

### LINKED ISSUE #179: Add deprecation warning
As indicated in https://github.com/smurfix/flask-script/issues/174#issuecomment-320324008 and from current activity, this project isn't actively maintained anymore. It would be nice to put some statement in docs and README so it is more obvious (probably with a point towards Flask's builtin CLI docs as a migration path).

### PR DESCRIPTION
**Pull request by notpushkin**
(Closes #179)

### CODE REVIEW

**Comment by iamale** [comment]
A little rough, but should do I guess?

**Comment by jeffwidman** [comment]
This is fine by me, but I'd want @smurfix / @anthraxx to sign off as well, especially since @anthraxx just did the hard work to ship a new release. They may want to tone down the warning a little to something like:

> While the maintainers are willing to merge PR's, they are not actively developing features. As of Flask 0.11, Flask includes a [built-in CLI tool](http://flask.pocoo.org/docs/dev/cli/), and that may fit your needs better.

File: README.rst [hunk 1]
```diff
@@ -1,3 +1,11 @@
+Deprecated
+------------
+
+This module is not actively maintained. Please, use `flask.cli`__ instead.
+
+__ http://flask.pocoo.org/docs/0.12/cli/
```
Comment (jeffwidman) [inline README.rst]: Can we point these links at the `dev` version? http://flask.pocoo.org/docs/dev/cli/

Alternatively we could point at `latest`, but personally I'd rather point at the dev version as that will be the most up to date and includes a big warning at the top saying that it's may be different than the latest stable release.

Comment (iamale) [reply README.rst]: Uhhh, sorry, forgot about `latest` (thought for some reason that it was specific to Read The Docs)

**Comment by smurfix** [comment]
@jeffwidman I agree with your wording. "Please use instead" is too strong IMHO.

File: README.rst (hunk 1 above)
Comment (iamale) [reply README.rst]: Also, `latest` seems a bit more appropriate here than `dev` as most people are probably using the stable version, and those on the development version should probably already know where to look for the docs.

**Comment by iamale** [comment]
You're right, it sounded a bit harsh. I've changed the wording as @jeffwidman suggested, but referring to “the maintainers” as a third-person (_they_) feels a bit weird, don't you think?

**Comment by anthraxx** [comment]
It would be better to use a non version specific URL as it will be outdated at a point.
A autó redirect is under http://flask.pocoo.org/docs/latest/cli/

**Comment by iamale** [comment]
Sorry, totally forgot I didn't push that one.

**Comment by jeffwidman** [comment]
Thank you!

**Pull request merged**